### PR TITLE
[es] Improved some Spanish translations

### PR DESCRIPTION
**Pull request by unknownbreakdown**

### CODE REVIEW

File: json/es.json [hunk 1]
```diff
@@ -5,45 +5,45 @@
   "click here to request another": "haga clic aquí para solicitar otro",
   "Confirm Password": "Confirmar contraseña",
   "Dashboard": "Panel",
-  "E-Mail Address": "Correo electrónico",
+  "Email Address": "Correo Electrónico",
```
[the remark this answers is not in the document]
Comment (unknownbreakdown) [reply json/es.json:8]: Ok.

[the remark this answers is not in the document]
Comment (unknownbreakdown) [reply json/es.json:8]: It's usual to start with uppercase all words on a label.

File: json/es.json [hunk 2]
```diff
@@ -5,45 +5,45 @@
...
   "Hello!": "¡Hola!",
   "If you did not create an account, no further action is required.": "Si no ha creado una cuenta, no se requiere ninguna acción adicional.",
   "If you did not receive the email": "Si no ha recibido el correo electrónico",
-  "If you did not request a password reset, no further action is required.": "Si no ha solicitado el restablecimiento de contraseña, omita este correo electrónico.",
+  "If you did not request a password reset, no further action is required.": "Si no ha solicitado el restablecimiento de contraseña, omita este mensaje de correo electrónico.",
   "If you’re having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "Si tiene problemas para hacer clic en el botón \":actionText\", copie y pegue la siguiente URL \nen su navegador web:",
   "Invalid signature.": "Firma no válida.",
-  "Login": "Entrar",
-  "Logout": "Salir",
+  "Login": "Iniciar Sesión",
```
[the remark this answers is not in the document]
Comment (unknownbreakdown) [reply json/es.json:18]: Ok. I just suggested a common way to label Login in Spanish.

File: json/es.json [hunk 3]
```diff
@@ -5,45 +5,45 @@
...
+  "Logout": "Finalizar Sesión",
```
[the remark this answers is not in the document]
Comment (unknownbreakdown) [reply json/es.json:19]: Ok. I just suggested a common way to label Logout in Spanish.

File: json/es.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (caouecs) [reply json/es.json:8]: Ok, but it's the only tag with uppercase ?

[the remark this answers is not in the document]
Comment (unknownbreakdown) [reply json/es.json:8]: I think, at this point you can discard my pull request.  It's not mandatory to uppercase all words on a label.

**Comment by caouecs** [comment]
Ok, I re open, see #1295 but... don't change tag "E-Mail Address" please, and we must finish for E of Electronico

and after no problem to merge

**Comment by caouecs** [comment]
Gracias

**Pull request merged**